src/web/event-notifications/event-notification-details/SlackNotificationDetails.jsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import {ReadOnlyFormGroup} from 'components/common';
+import {Well} from 'components/graylog';
```

Comment:
Something is up with the eslint config on this repo because there are lots of lint errors I can spot

Reply:
I think, there's is no eslint configured with this jenkins build currently.